### PR TITLE
switch to https

### PR DESCRIPTION
**Pull request by cajaks2**
This should 100% be https

### CODE REVIEW

**Comment by netmanchris** [comment]
@cajaks2  Just getting back to this library after some time away and some tests are failing. My guess is that your PR failing test are due to faults in the test and not your code. Looking into fixing test now. 

Will review the PR once I get the tests resolved.

**Review by netmanchris — CHANGES_REQUESTED** [review]
Cut and paste added an extra s in a couple of places. Please take a quick look and make sure that https:// is valid in all changes, including in the Docstrings.  Other than that, looks good and I was able to get through a Travis run without server-side errors so code is still functional based on tests.

File: pyawair/devices.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def get_dev_timezone(auth, device_name=None, device_type=None, device_id=None):
     """
     Function to get the timezone for a single specific devices for the account
     linked to the token. Refer to column TZ in
-    https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+    httpss://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```
Comment (netmanchris) [inline pyawair/devices.py]: Extra "s" in url header

File: pyawair/devices.py [hunk 2]
```diff
@@ -245,7 +245,7 @@ def set_device_timezone(auth, timezone, device_name=None, device_type=None, devi
     :param auth:
     :param timezone: str value of the desired timezone.
     For specific values please see
-    https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+    httpss://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```
Comment (netmanchris) [inline pyawair/devices.py]: Extra "s" in url header

**Comment by netmanchris** [comment]
@cajaks2  Can you please merge in the test fixes into your PR so that we can get the Travis build to run again? thanks!

**Comment by cajaks2** [comment]
Should be good at this point: LED tests also seemed to fail with a 502.

**Comment by netmanchris** [comment]
Talked to Awair and there are some flacky tests due to what's probably an issue on their side. ( 502 points to that being the case ). Tests passed so I'll merge this through and watch in the future to see how things go. Thanks!

**Pull request merged**